docs/standards/javascript_standards.md

```diff
@@ -2,13 +2,27 @@
 
 Use vanilla JavaScript, don't use extensions to the language such as TypeScript
 
-### Use [Standard JS](https://standardjs.com/) to lint your code
+### Use [neostandard](https://github.com/neostandard/neostandard) to lint your code
+
+neostandard is an opinioned JavaScript linter that enforces a consistent coding style.
```

Comment:
Do you mean opinionated instead of opinioned?

Reply:
Yes I do!